app/views/users/_edit.haml

```diff
@@ -176,9 +176,66 @@
                   = type.check_box :contacts_birthday, {checked: email_prefs["contacts_birthday"]}, false, true
                   = t(".birthday")
                 .small-horizontal-spacer
+      .col-md-6
+        %h3
+          = t(".receive_app_notifications")
+        = form_for "user", url: edit_user_path, html: {method: :put} do |f|
+          = f.fields_for :email_preferences do |type|
+            #email_prefs
```

Comment:
- Do not repeat id "#email_prefs" on the page
- `id` attribute must be in lisp-case